### DISCUSSION TITLE
Drittgutachter / Dissertation

### DISCUSSION THREAD

**Discussion by SpalterU** [tud-cd/tudscr#72] [Q&A]
Ich schreibe gerade an meiner Diss. zu welcher ich formal drei Gutachter habe (MaschWesen Promotionsordnung 2001). In dem LaTex-Dokument kann ich jedoch  mit  "\referee{ Max \and Moritz \and Bolde} nur zwei Gutachter sauber einfügen (also mit der Auflistung "Erstgutachter, Zweitgutachter"}. Der dritte Position wird einfach unter der zweiten gesetzt, jedoch ohne die Auflistung "Drittgutachter". Gibt es hier eine Möglichkeit das zu ermöglichen?
LaTEx-File:
"\documentclass[%
  english,ngerman,%
  BCOR=6mm,cdgeometry=no,DIV=13%
]{tudscrreprt}
\iftutex
  \usepackage{fontspec}
\else
  \usepackage[T1]{fontenc}
  \usepackage[ngerman=ngerman-x-latest]{hyphsubst}
\fi

\input{./DokEinstellungen/Praeambel-210823}

\usepackage{babel}
\usepackage{isodate}
\usepackage{blindtext}
...
\thesis{diss}
...

**Comment by mrpiggi** [comment]
There is already a simplified solution in #38 but in order to create a dedicated caption for a third referee, this should work although I didn't test for any side effects:
```tex
\documentclass[ngerman,cdfont=false]{tudscrreprt}
\usepackage{babel}
\usepackage[T1]{fontenc}
\usepackage{isodate}

\begin{document}
\faculty{Juristische Fakultät}
\department{Fachrichtung Strafrecht}
\institute{Institut für Kriminologie}
\chair{Lehrstuhl für Kriminalprognose}
\date{18.02.2015}
\title{%
  Entwicklung eines optimalen Verfahrens zur Eroberung des
  Geldspeichers in Entenhausen
}
\subject{master} 
\graduation[M.Sc.]{Master of Science} 
\author{%
Mickey Mouse%
\matriculationnumber{12345678}%
\dateofbirth{2.1.1990}%
\placeofbirth{Dresden}%
}

\newrobustcmd*\refereehack{%
  \vskip 1ex plus .5ex minus .5ex%
  {\usekomafont{titlepage}{Drittgutachter}\par}
}
\referee{AA \and BB \refereehack CC}

\maketitle
\end{document}
```
I'm going to provide a more generic solution for the next version where `\refereeothername` and similiar commands can be provided as a comma separated list.
